Doc/library/typing.rst

```diff
@@ -49,8 +49,8 @@ annotations. These include:
      *Introducing* :class:`Protocol` and the
      :func:`@runtime_checkable<runtime_checkable>` decorator
 * :pep:`585`: Type Hinting Generics In Standard Collections
-     *Introducing* the ability to use builtin collections and ABCs as
-     :term:`generic types<generic type>`
+     *Introducing* :class:`types.GenericAlias` and the ability to use standard
+     library classes as :ref:`generic types<types-genericalias>`
```

Comment:
Wait, why'd the link change from the one in the glossary to about `types-genericalias`?

Reply:
My reasoning was:

- I think the `types-genericalias` link is more informative about the change that was introduced in the PEP.
- I think it's an important piece of documentation that deserves a prominent mention near the top of the `typing` docs.
- It's consistent with the link to `types-union` in the bullet point about PEP 604.

But I'm very happy to revert this change if you think the link to the glossary is more appropriate here 🙂